### DISCUSSION TITLE
Alist+189cloudpc+foldersync/autosync 同步出现错误

### DISCUSSION THREAD

**Discussion by shandongtlb** [AlistGo/alist#5027] [Q&A] [answered]
系统：Ubuntu22.04
alist版本：3.25.1，3.16+
出现问题：使用autosync等安卓app同步相册之后，过一段时间（1-2小时之后）同步下载的时候出现“下载失败”；这里特别说明一下，上传完毕立即同步不会出现该问题，1-2小时之后，再次点击同步会复现。
分析：通过调取autosync、foldersync等app的日志，发现是因为远程文件发生了改变，导致触发同步下载，又因为本地已经存在文件，因此会导致“冲突”，下载失败。

今年3月份左右的时候曾经使用过该组合（Alist+189cloudpc+foldersync）一段时间，未发现问题，于是我使用当时的版本进行测试，发现问题仍然复现。

然而使用Alist+阿里云盘open不会出现该问题，只有天翼云盘，因此猜测大概率是天翼云服务器将文件进行了修改。有没有巨佬来解答一下？

贴一下配置：
由于现在我决定使用alistv2试试能不能复现，需要等一个多小时，因此配置简单说一下吧先：

一切都是默认配置，天翼pc驱动，网盘根目录挂载到alist根目录，关闭web代理，开启302代理，并且上传方式无论是旧版、流式还是秒传，均会复现问题。

**Comment by shandongtlb** [comment]
如图
![Screenshot_20230817_163557_Autosync](https://github.com/alist-org/alist/assets/40312509/f0c308b0-867b-47f5-913a-b78caa34ae1e)

最新版本，当前已修改webdav302为本地代理，已经不报错了，但是会覆盖本地文件，因此问题依然存在。

**Comment by foxxorcat** [comment, marked as answer]
302是软件不支持。

一般情况下同步会对比修改时间，文件大小，hash值。
使用webdav hash值应该不会对比，那对比的就只有文件大小和修改时间了。
然后天翼云盘pc修改时间默认为上传时间，所以有可能是这个问题。（当然也可能真变了，你可以手动下载计算hash对比）
解决方法：更改同步策略为只对比大小或换软件

**Reply by shandongtlb** [reply]
好的大佬，我研究一下，谢谢啦！
